### ISSUE TITLE
Links on expandable tile on hover have insufficient color contrast

### ISSUE THREAD

**Issue by tessarodes** [carbon-design-system/carbon#8701]
Hi Carbon love you guys thanks for all you do.

Anyway! I know the below is not the ideal usage of an expandable tile, but I'm reviewing a lot of services lately that use a really, really long expanded state, which means hovering the entire time is unavoidable, and they include learn more links throughout, which I think should be allowed. However, the link color isn't accessible on the tile's hover color:

![image](https://user-images.githubusercontent.com/10426703/118525842-3e20f980-b705-11eb-99ac-924e885aa729.png)
![image](https://user-images.githubusercontent.com/10426703/118525797-33fefb00-b705-11eb-8728-c965dbf53118.png)

Is there a way we can help the services implement this better? Can you optionally disable the background color change hoverstate for this tile ever? Or give the link a different color on hover?

**Comment by tw15egan** [comment]
@aagonzales I believe we created `$link-secondary` for similar situations in `DataTable`, perhaps we just need to set any `Link` elements inside `ExpandableTile` to `link-secondary when the tile is hovered?
